profiles/arch/arm64/package.use.mask

```diff
@@ -435,3 +435,7 @@ sci-libs/linux-gpib firmware
 # Mask flags of calligra and dependencies due to lack of keywords
 app-office/calligra spacenav
 media-gfx/pstoedit emf
+
+# Jon Janzen <jon@jonjanzen.com> (2023-09-11)
+# librttopo has not been tested on this arch
+dev-db/spatializte rttopo
```

Comment:
I'm a bit confused why we still get the `NonsolvableDepsInStable` error on `arm64`, the flag is masked so it should work.

Reply:
> I'm a bit confused why we still get the `NonsolvableDepsInStable` error on `arm64`, the flag is masked so it should work.

aaah, it's because there is a type: `dev-db/spatializte` -> `dev-db/spatialite`.